js/anims-missiles.js

```diff
@@ -0,0 +1,111 @@
+//global scope
+
+var columnID = 0
+
+
+var gameScore = {
+    score:0,
+    toll:0
+}
+
+setTimeout(repeatAnims, 500)
+repeatAnims()
+
+function repeatAnims() {
+    setInterval(function() {
+        dropMissile(2000)
+    }, 1000)
+}
+
+function dropMissile(animationTime) {
+    var columnNumber = randomNumGen(4)
+    var $bomb = $('<img src="img/missile-drop-down-white.png" ' +
+        'class="missile bomb-'+ columnNumber +'">')
+    $('#pos-'+columnNumber).prepend($bomb)
+
+    $bomb.animate({
+        top: "+=550"
+    }, animationTime, "linear", function () {
+        missileHitsCity()
+        $(this).remove()
+        document.querySelector('#toll').innerHTML = incrementToll()
+    })
+}
+
+function missileHitsCity() {
+    gameScore.score -= 1
+    gameScore.toll += randomNumGen(10000)
```

Comment:
Random number means you can't use it as a lives system, if we were to implement that in future.

Reply:
have made it a constant 10,000 per miss.